From 8a8647a0396090a87844a571ac003b0f10044e1d Mon Sep 17 00:00:00 2001
From: "lauren n. liberda" <lauren@selfisekai.rocks>
Date: Wed, 19 Jun 2024 23:00:32 +0200
Subject: [PATCH] unbundle: add missing icu target

solves "ERROR Unresolved dependencies.
//chrome/browser/ui/lens:unit_tests(//build/toolchain/linux/unbundle:default)
  needs //third_party/icu:icuuc_public(//build/toolchain/linux/unbundle:default)"
---
 build/linux/unbundle/icu.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/linux/unbundle/icu.gn b/build/linux/unbundle/icu.gn
index 366ffd5a3597e..bad83cda0283e 100644
--- a/build/linux/unbundle/icu.gn
+++ b/build/linux/unbundle/icu.gn
@@ -61,6 +61,10 @@ source_set("icuuc") {
   ]
 }
 
+group("icuuc_public") {
+  public_deps = [ ":icuuc" ]
+}
+
 shim_headers("icui18n_shim") {
   root_path = "source/i18n"
   headers = [
